Fortnox.NET/Fortnox.NET.csproj

```diff
@@ -21,6 +21,11 @@
         <StartupObject />
     </PropertyGroup>
 
+    <PropertyGroup>
+      <GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <NoWarn>$(NoWarn);1591</NoWarn> <!-- // NOTE(Oskar): Disables the warning "Missing XML comment for publicly visible type or member". -->
```

Comment:
is this all over the place, or could we do it on a case-to-case basis to better show where docs are needed and where we intentionally waited with it?

Reply:
Its hiding the warning all over the place since theres no documentation for most methods yet. I think a more organized effort can be made to add this documentation in form of issues here on github or something similar.

Reply:
ok, makes sense